src/loaders/external_jpg/meson.build

```diff
@@ -0,0 +1,18 @@
+source_file = [
+   'tvgJpgLoader.h',
+   'tvgJpgLoader.cpp',
+]
+
+jpg_dep = dependency('libturbojpeg', required: false)
+
+if not jpg_dep.found()
+	jpg_dep = cc.find_library('turbojpeg', required: false)
+endif
```

Comment:
I think this dependency check can be handled by the parent build script.

Reply:
This dependency must be addressed here so that external_jpg can be isolated itself.
That's why I left it. Otherwise parent <-> external_jpg have the strong coupling.